tests/hotplug/memory.go

```diff
@@ -414,3 +428,13 @@ func getVMIMigration(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineIn
 	}, 30*time.Second, time.Second).Should(BeTrue(), "A migration should be created")
 	return migration
 }
+
+// The VMI is assumed to be already logged-in.
+func getGuestMemory(vmi *v1.VirtualMachineInstance) *resource.Quantity {
```

Comment:
Is there an advantage for using `*resource.Quantity` over a simple int?
Additionally, please consider returning an error and asserting that it didn't occur outside of the function.

Reply:
> Is there an advantage for using *resource.Quantity over a simple int?

Yes. This object is both capable of doing certain operations for you, like comparing (e.g. with different units, like GBs etc) and presenting as string.
In general, I want an object that represents the amount of memory that the VM has. I don't really care if it's measured in bytes or kilobytes behind the scenes.

> Additionally, please consider returning an error and asserting that it didn't occur outside of the function.

Sure, done.